utils/check_docstrings.py

```diff
@@ -653,6 +653,8 @@
     "TFRagModel",
     "TFRagSequenceForGeneration",
     "TFRagTokenForGeneration",
+    "TFRegNetModel",
+    "TFRegNetForImageClassification",
```

Comment:
Agreed. These lines were added because their omission was causing this check to fail when running make fixup. I'm not sure how that check is passing elsewhere without this, but we can remove these changes from this PR is that's more appropriate.

Reply:
```suggestion
```

Reply:
@amyeroberts I ran some tests and I think everything is ready to go except for this last change. The checks have passed and the gradient checkpointing is working now, so with this last commit I think it may be ready for merge, let me know what you think.